src/common/meta/src/kv_backend/memory.rs

```diff
@@ -96,6 +96,14 @@ impl<T: ErrorExt + Send + Sync + 'static> KvBackend for MemoryKvBackend<T> {
 
         let iter: Box<dyn Iterator<Item = (&Vec<u8>, &Vec<u8>)>> = if range_end.is_empty() {
             Box::new(kvs.get_key_value(&key).into_iter())
+        } else if range_end.len() == 1 && range_end[0] == 0 {
+            // If both key and range_end are ‘\0’, then range represents all keys.
+            if key.len() == 1 && key[0] == 0 {
+                Box::new(kvs.iter())
+            } else {
+                // If range_end is ‘\0’, the range is all keys greater than or equal to the key argument.
+                Box::new(kvs.range(key..))
+            }
```

Comment:
Does the `RaftEngineBackend` also need to be synchronized?

Reply:
Yes, and we also need to fix the `delete_range`.